### PR TITLE
Custom elements upgrade

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For Starcounter/RebelsLounge#192

Caution! There are tests failing, but It's not related to this PR. I've run build on Teamcity with current develop branch and result was the same - tests were failing.

### CODE REVIEW

**Comment by PatrykSzwer** [comment]
I see that after merge there are still some tests failing. I'll investigate this further.

**Comment by PatrykSzwer** [comment]
Looks like again somethings happening with build server. Locally all tests are passing. But on the server, when I try to run any tests for kitchensink I'm getting errors like:
image

**Comment by warpech** [comment]
We should not merge until the branch builds in TeamCity

**Comment by PatrykSzwer** [comment]
After restarting agent, everything is working fine. All tests passed.

File: src/KitchenSink/wwwroot/sys/juicy-select/.bower.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer"
+    "polymer": "Polymer/polymer#1.9 - 2"
   },
-  "_release": "9e6e0be17b",
```
Comment (warpech) [inline src/KitchenSink/wwwroot/sys/juicy-select/.bower.json]: Please change in `src/KitchenSink/bower.json` to use juicy-select 1.0.0 instead of `master`

**Comment by warpech** [comment]
`google-map` is still in version 1.2.0. Should be 2.0.2

**Comment by warpech** [comment]
`hot-table` will need a few external fixes (tracked in https://github.com/handsontable/hot-table/issues/44). This can be a separate issue.

**Comment by warpech** [comment]
`iron-selector` is in version 1.5.3. Should be 2.0.0

**Comment by warpech** [comment]
[starcounter-upload](https://github.com/Starcounter/starcounter-upload) is at version 0.0.7. Please release a version 1.0.0 and use it here

**Comment by warpech** [comment]
[pikaday-decorator](https://github.com/display-interactive/pikaday-decorator) is at version 0.2.1. Current version is 0.8.3, but it still does not use `<slot></slot>` element. We will need to find another datepicker or make a PR to `pickaday-decorator`, whatever is easier.

File: src/KitchenSink/wwwroot/sys/juicy-select/.bower.json (hunk 1 above)
Comment (PatrykSzwer) [reply src/KitchenSink/wwwroot/sys/juicy-select/.bower.json]: Done

**Comment by PatrykSzwer** [comment]
> google-map is still in version 1.2.0. Should be 2.0.2

Done
> There is one folder in /sys which is completely unused and not present in bower.json: iron-collapse. Please remove it.

Done
> iron-selector is in version 1.5.3. Should be 2.0.0

Done

**Comment by warpech** [comment]
Note to self: @PatrykSzwer went on vacation. He says that these tests need improvement or temporary ignore in Firefox. These tests are passing if you have remote desktop open and are looking at it :)

**Comment by warpech** [comment]
I created a new issue for the flaky tests: #181

**Pull request merged**